### ISSUE TITLE
Next meetup

### ISSUE THREAD

**Issue by frankcash** [nodeschool/tampa#1]

**Comment by njj** [comment]
@eliwheaton is this still a thing?  I'm moving back to Tampa in a few months.

**Comment by eliwheaton** [comment]
@njj yeah, join the [meetup for updates](http://www.meetup.com/tampa-nodejs)

**Comment by julianduque** [comment]
@eliwheaton do you have access to a venue for the next NodeSchool? I can sponsor food and drinks and mentor de event but since I'm new around, I don't have an idea or know nobody with a venue we can use.

**Comment by njj** [comment]
I'm interested in attending this! @julianduque

**Comment by eliwheaton** [comment]
@julianduque we do have access to a venue, but it's only for a couple hours at night, which works for meetups but not so great for a nodeschool event. I've been trying to find a better venue for that, possibly one available on a weekend so we can meet during the day.

**Comment by njj** [comment]
@eliwheaton I have an office Ybor City that has a decent size conference room, I can talk to the property managers about utilizing it.

**Comment by julianduque** [comment]
@njj nice! thats great!

@eliwheaton I've been talking with The Iron Yard (on Suncoast Dev Guild) and it seems we will have a space on the next Startup Week in Tampa

**Comment by julianduque** [comment]
Here is the event  http://tampabaystartupweek2016.sched.org/event/60Bj/nodeschool-learn-nodejs-with-open-source-workshops?iframe=no

**Comment by eliwheaton** [comment]
@julianduque awesome! I posted it on the meetup page.

@njj That'd be great, let us know what they say. How many people can it hold?
